### PR TITLE
Add keybinding to toggle twitch iframe display

### PR DESCRIPTION
**Pull request by ImFlog**
See https://github.com/nugetchar/sidetwitch/issues/5

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -46,47 +48,72 @@ chrome.tabs.onActivated.addListener(function(tabId, changeInfo, tab) {
...
-      if (!!tabs[0]) {
-        console.info('We hope you\'re having a good time');
-        chrome.tabs.sendMessage(
-            tabs[0].id,
-            {type: createType, text: currentChannel},
-            () => {}
-        )
-      }
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+        if (!!tabs[0]) {
```
Comment (thomastoledo) [inline js/background.js:52]: We're going to need to start preventing the "tabs undefined" errors in the future. I won't refuse the PR but I'll think of a way to set up a dev environment so we can use TypeScript for more safety and quality in our code.

**Review by thomastoledo — COMMENTED** [review]
Seems good to me, going to test ASAP

**Comment by ImFlog** [comment]
@nugetchar Should we add an UI hint about this feature ?
In the popup maybe ?

**Comment by thomastoledo** [comment]
@ImFlog Agree. I'll accept this PR and we'll do another one for this hint

**Pull request merged**